### PR TITLE
Disabled volume create and delete command in mayactl

### PR DESCRIPTION
**Pull request by ashishranjan738**
Signed-off-by: Ashish Ranjan <ashishranjan738@gmail.com>

**What this PR does / why we need it**:
->     Disabled volume create and delete command in mayactl

**Special notes for your reviewer**:
-> Will enable it once api starts working .

### CODE REVIEW

File: cmd/mayactl/app/command/volume.go [hunk 1]
```diff
@@ -107,9 +108,9 @@ func NewCmdVolume() *cobra.Command {
 	}
 
 	cmd.AddCommand(
-		NewCmdVolumeCreate(),
+		// NewCmdVolumeCreate(),
 		NewCmdVolumesList(),
-		NewCmdVolumeDelete(),
+		// NewCmdVolumeDelete(),
```
Comment (kmova) [inline cmd/mayactl/app/command/volume.go]: Delete works .. @ashishranjan738

File: cmd/mayactl/app/command/volume.go [hunk 2]
```diff
@@ -61,26 +61,19 @@ const (
...
    $ mayactl volume list
 
- # Delete a Volume:
-   $ mayactl volume delete --volname <vol>
-
- # Delete a Volume created in 'test' namespace:
-   $ mayactl volume delete --volname <vol> --namespace test
-
  # Statistics of a Volume:
    $ mayactl volume stats --volname <vol>
```
Comment (kmova) [inline cmd/mayactl/app/command/volume.go]: add namespace here.

Comment (kmova) [reply cmd/mayactl/app/command/volume.go]: my bad. I see it is already there.

Comment (ashishranjan738) [reply cmd/mayactl/app/command/volume.go]: yup

**Pull request merged**